firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/DataCollectionArbiter.java

```diff
@@ -71,6 +71,10 @@ public DataCollectionArbiter(FirebaseApp app) {
     }
   }
 
+  public synchronized boolean isCrashlyticsCollectionEnabled() {
+    return isAutomaticDataCollectionEnabled();
```

Comment:
Can you call this directly from CrashlyticsCore instead? If the visibility is private you can make it package-private (default).